cookbook/frontend/assetic_php.rst

```diff
@@ -0,0 +1,152 @@
+.. index::
+    single: Front-end; Assetic, Bootstrap
+
+Combining, Compiling and Minimizing Web Assets with PHP Libraries
+=================================================================
+
+The official Symfony Best Practices recommend to use Assetic to
+:doc:`manage web assets </best_practices/web-assets>`, unless you are
+comfortable with JavaScript-based frontend tools.
+
+Even if those JavaScript-based solutions are the most suitable ones from a
+technical point of view, using pure PHP alternative libraries can be useful in
+some scenarios:
+
+* If you can't install or use ``npm`` and the other JavaScript solutions;
+* If you prefer to limit the amount of different technologies used in your
+  applications;
+* If you want to simplify application deployment.
+
+In this article you'll learn how to combine and minimize CSS and JavaScript files
+and how to compile Sass SCSS files using PHP only libraries with Assetic.
+
+Installing the Third-Party Compression Libraries
+------------------------------------------------
+
+Assetic includes a lot of ready-to-use filters but it doesn't include their
+associated libraries. Therefore, before enabling the filters used in this article
+you must install two libraries. Open a command console, browse to your project
+directory and execute the following commands:
+
+.. code-block:: bash
+
+    $ composer require leafo/scssphp
+    $ composer require patchwork/jsqueeze:"~1.0"
+
+It's very important to maintain the ``~1.0`` version constraint for the ``jsqueeze``
+dependency because the most recent stable version is not compatible with Assetic.
+
+Organizing Your Web Asset Files
+-------------------------------
+
+This example shows the very common scenario of using the Bootstrap framework, the
```

Comment:
-1 for adding "very common". It's starting to look like Symfony is sponsered by Bootstrap :)